### DISCUSSION TITLE
Extension disable and JavaScript exucution

### DISCUSSION THREAD

**Discussion by RacoonyCoon** [3kh0/ext-remover#1363] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi recently my school removed ad blockers such as ublock origin so I am not able to execute JavaScript then I moved on to rigtools witch was amazing I could disable iboss and my lockdown browser for schoolcity however one day it says blocked my administrator. I just want the ability to execute JavaScript and possibly disable extenstions however hanging freezing or anything doesnt work for me. I don't know if there are alot of exploits that can help because our school has ASU students block our stuff and figure out how to block everything I still believe there could be a way however so if anyone could please help please do so.

**Comment by Charlieis1nsan3** [comment]
Use Snap&Run, but also check your chrome://os-settings and search for networking, and see if your network allows, DNS editing, and maybe you can do DNS bypassing such as omada.cafe, or caub.

**Reply by RacoonyCoon** [reply, marked as answer]
I will try the other stuff you said but Snap&Run is blocked by administer like basically everything besides a few they allow
